api/app/models/annotation.rb

```diff
@@ -357,5 +374,17 @@ def build_by_reading_group_membership_scope(rgm)
 
       includes(:membership_comments).where(condition)
     end
+
+    def build_keyword_scope(value)
+      escaped = value.gsub("%", "\\%")
+
+      needle = "%#{escaped}%"
+
+      body_matches = where arel_table[:body].matches(needle)
+
+      creator_matches = joins(:creator).where(User.arel_table[:first_name].matches(needle).or(User.arel_table[:last_name].matches(needle)))
+
+      creator_matches.or(body_matches).distinct
+    end
```

Comment:
In case anyone looks at this while I'm out— this is the main piece of this work that I know needs to change. I'm fairly certain this way of setting up a keyword scope won't coexist with the pre-existing search. Annotations are indexed for search, but there are various conditions around visibility that exclude annotations from search that we don't want to apply in the admin view. In addition, for the admin list search, I don't think we need full text indexing of the annotation subject, etc. A simple keyword match on a couple of fields like creator name and body seems likely to get the results users are after.